api/answers/views.py

```diff
@@ -18,58 +18,66 @@
 
 
 class AnswersPostView(APIView):
-    """
+    '''
     Views for posting an answer
-    """
+    '''
     permission_classes = [permissions.IsAuthenticated]
 
-    def post(self, request, id, meetupId):
-        """
+    def post(self, request, questionId, meetupId):
+        '''
         Post an answer
-        """
+        '''
         try:
             queryset = Meetup.objects.get(id=meetupId)
-        except ValidationError:
-            return Response(
-                {
-                    "error": "Meetup doesn't exist"
-                },
-                status=status.HTTP_404_NOT_FOUND
-            )
-
-        try:
-            queryset = Question.objects.all()
-            request.data['creator'] = request.user
-            serializer = AnswerSerializer(data=request.data)
-            answer_body = Answer.objects.filter(body=request.data.get('body'))
+            try:
+                queryset = Question.objects.all()
+                request.data['creator'] = request.user
+                serializer = AnswerSerializer(data=request.data)
+                answer_body = Answer.objects.filter(
+                    body=request.data.get('body'))
 
-            if serializer.is_valid():
-                if answer_body:
+                if valid_string(request.data.get('body')) == False:
                     return Response(
                         {
-                            "error": "Answer already exist"
+                            'error': 'Please enter a valid answer'
                         },
                         status=status.HTTP_400_BAD_REQUEST
                     )
-                Answer.objects.update_or_create(
-                    body=request.data.get('body').strip(),
-                    defaults={
-                        'question': get_object_or_404(queryset, id=id),
-                        'creator': request.data.get('creator')
-                    }
+
```

Comment:
Hello @Bakley ,
This is good work,
Would you try to avoid many return statements per method or function(One entry point and one exit point philosophy), it would make it better.
Look at this short article that speaks about the same [Why many return statements are bad](https://www.yegor256.com/2015/08/18/multiple-return-statements-in-oop.html)

Reply:
Thank you @Ogutu-Brian